x/profiles/types/msgs_relationships.go

```diff
@@ -41,10 +41,6 @@ func (msg MsgCreateRelationship) ValidateBasic() error {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "sender and receiver must be different")
 	}
 
-	if !IsValidSubspace(msg.Subspace) {
```

Comment:
I think the subspace should not be 0 since subspace 0 is the dummy subspace used for invalid error.

Reply:
Subspace with ID `0` is not actually used as a dummy subspace for invalid errors. It's the subspace that identifies the entire set of subspaces. So for example if someone blocks you on subspace `0` they are blocking you on **all** subspaces. We could call it the "Desmos" subspace.